modules/is/manifests/params.pp

```diff
@@ -35,6 +35,44 @@
 
   $start_script_template = 'bin/wso2server.sh'
 
+  $template_list = [
+    'repository/conf/carbon.xml',
+    'repository/conf/user-mgt.xml',
+    'repository/conf/datasources/master-datasources.xml',
+    'repository/conf/axis2/axis2.xml',
+    'repository/conf/identity/identity.xml',
+  ]
+
+  # carbon.xml configs
+  $ports_offset = 0
+  /*
+     Host name or IP address of the machine hosting this server
+     e.g. www.wso2.org, 192.168.1.10
+     This is will become part of the End Point Reference of the
+     services deployed on this server instance.
+  */
+  $hostname = 'localhost'
+  $mgt_hostname = 'localhost'
+
+  $security_keystore_location = '${carbon.home}/repository/resources/security/wso2carbon.jks'
```

Comment:
Shouldn't these keystore configurations go to common module since this is common to all modules in common scenarios?